src/adapter/src/coord.rs

```diff
@@ -763,10 +769,6 @@ impl<S: Append + 'static> Coordinator<S> {
             let _enter = span.enter();
 
             self.handle_message(msg).await;
-
-            if let Some(timestamp) = self.get_local_timestamp_oracle_mut().should_advance_to() {
-                self.queue_local_input_advances(timestamp).await;
-            }
```

Comment:
By removing this, do we risk making the table advancement a low priority event? Seems removing this takes it from "once per message received" to "lowest priority event".

Reply:
Queuing a table advancement was happening on a "once per message received" rate. However, actually executing the table advancements was already the lowest priority message that the Coordinator handled.

In practice the only things that triggered this were group commit, DDL, and the `advance_timelines_interval`, all of which now explicitly advance tables.